sympy/printing/mathml.py

```diff
@@ -13,24 +13,37 @@
 from .conventions import split_super_sub, requires_partial
 
 
-class MathMLPrinter(Printer):
-    """Prints an expression to the MathML markup language
+class MathMLPrinterBase(Printer):
+    """Contains common code required for MathMLContentPrinter and
 
```

Comment:
Unnecessary newline.

Reply:
Should I remove newlines in other docstrings also like from `MathMLContentPrinter` and `MathMLPresentationPrinter`. I used newline because docstrings previously were written in this format .

Reply:
@Subhash-Saurabh There is no need to put a newline within sentences. Between paragraphs and sections are okay.